src/utilities/time.test.ts

```diff
@@ -40,6 +41,23 @@ test('convertDurationStringToUs', () => {
     `);
 });
 
+test('convertUTCtoMs', () => {
+  // standard date conversion
+  expect(convertUTCtoMs('2024-01-01T00:00:00Z')).toEqual(1704067200000);
+
+  // DOY doesn't work
+  expect(convertUTCtoMs('2024-001T00:00:00Z')).toEqual(NaN);
+
+  // conversion to DOY is fine if the time zone ("Z") is excluded
+  expect(convertUTCtoMs(convertDoyToYmd('2024-001T00:00:00') ?? '')).toEqual(1704067200000);
+
+  // conversion without a timezone in the input
+  expect(convertUTCtoMs('2024-01-01 00:00:00')).toEqual(new Date('2024-01-01 00:00:00').getTime());
```

Comment:
Can you add a comment explaining that the expected value needs to utilize the test runner machine's time zone in order pass, thus a new Date object is being used?